cetmix_tower_server/views/cx_tower_plan_log_view.xml

```diff
@@ -88,70 +89,89 @@
+            <filter
                     string="Success"
                     name="filter_success"
                     domain="[('plan_status', '=', 0)]"
                 />
-                <filter
+            <filter
                     string="Error"
                     name="filter_error"
                     domain="[('plan_status', '!=', 0)]"
                 />
-                <filter
+            <filter
                     string="Running Now"
                     name="filter_is_running"
                     domain="[('is_running', '=', True)]"
                 />
-                <separator />
-                <filter
+            <separator />
+            <filter
                     string="Labeled"
                     name="filter_labeled"
                     domain="[('label', '!=', False)]"
                 />
-                <group expand="0" string="Group By">
-                    <filter
+            <filter
+                    string="Main plans"
+                    name="filter_main_plans"
+                    domain="[('parent_flight_plan_log_id', '=', False)]"
+                />
+            <filter
+                    string="Child plans"
+                    name="filter_child_plans"
+                    domain="[('parent_flight_plan_log_id', '!=', False)]"
+                />
+            <separator />
+            <group expand="0" string="Group By">
+                <filter
                         string="Server"
                         name="group_server"
                         domain="[]"
                         context="{'group_by': 'server_id'}"
                     />
-                    <filter
+                <filter
                         string="Flight Plan"
                         name="group_plan"
                         domain="[]"
                         context="{'group_by': 'plan_id'}"
                     />
-                    <filter
+                <filter
                         string="Start date"
                         name="group_start"
                         domain="[]"
                         context="{'group_by': 'start_date:day'}"
                     />
-                    <filter
+                <filter
                         string="Finish date"
                         name="group_finish"
                         domain="[]"
                         context="{'group_by': 'finish_date:day'}"
                     />
-                </group>
-            </search>
-        </field>
-    </record>
+                <filter
+                        string="Main plan"
+                        name="group_main_plan"
+                        domain="[]"
+                        context="{'group_by': 'parent_flight_plan_log_id'}"
+                    />
+            </group>
+        </search>
+    </field>
+</record>
+
 
     <record id="action_cx_tower_plan_log" model="ir.actions.act_window">
         <field name="name">Flight Plan Log</field>
```

Comment:
This filter should be enabled by default:
![image](https://github.com/user-attachments/assets/b8ed6665-304c-479b-91ae-fb120733287c)